mer_verify_kernel_config

```diff
@@ -262,6 +262,7 @@ CONFIG_HIDRAW			y,m,!	# optional: Support HID devices
 CONFIG_UNIX			y	# UNIX sockets option is required to run Mer
 CONFIG_SYSVIPC			y	# Inter Process Communication option is required to run Mer
 CONFIG_EXT4_FS			y,m,!	# Mer uses ext4 as rootfs by default
+CONFIG_CUSE			y	# required for CUSE support.
```

Comment:
Should we have y,! here so it is recommended but not failing build if it is missing as it is not directly required?

Reply:
No, we expect that all new kernels will have that feature.

Reply:
Yeah, but everybody does not have new enough kernel with CUSE config who is porting the device for Sailfish

Reply:
I would say they have since its there since Linux 2.6.31:
https://cateee.net/lkddb/web-lkddb/CUSE.html

Reply:
In any case you could require it only starting with a kernel version.

Reply:
Fixed